_posts/2019-03-04-jumping-on-the-dash-less-bandwagon.markdown

```diff
@@ -0,0 +1,64 @@
+title:  "Jumping on the '-less' Bandwagon"
+date:   2019-03-04
+tags: serverless
+author: Pete Yandell
+image: img/bandwagonless/bandwagonless.png
+---
+
+<img style="display: block; margin-left: auto; margin-right: auto; width: 40%;" alt="wagon wheel" src="{{ site.url }}/img/bandwagonless/bandwagonless.png"/>
+
+## Introduction
+
+It has been awhile since I made my [last post](https://mechanicalrock.github.io/2018/08/27/inception-pipelines-pt6), the one where at the end I promised something about the [AWS CDK](https://awslabs.github.io/aws-cdk/) and how awesome it was going to be. Since then a few of my colleagues have mastered it, and I got super excited by something else.
+
+That something else is SER-VER-LESS!! And so now, I to, am _Jumping on the '-less' Bandwagon!_
+
+## What is serverless and why all the hype?
+
+If you scour the Internet, you will find as many definitions as there are search results. Most of them dive into esoteric technical definitions of what is and isn’t serverless.
+
+My technical perspective is that I don’t consider building, running and managing containers as serverless. I also don't count no-infrastructure pieces of your architecture like DynamoDB, Identity management, payment gateways, etc as Serverless either. They are however critical ‘no-ops’ pieces to operating Serverlessly. What’s left is really the functions-as-a-service services like AWS Lambda, Google Functions and Azure Functions. These functions strip away everything that are not critical core features that are unique to your business (versus structural piece like databases)
+
+Outside the technical realm, Serverless is also a mindset that flows right back into the core how your business operates. In the next section, I’ll discuss why fully embracing serverless very likely means a shakeup of how IT operates. All traditional IT roles come into question when you consider how much of the day-to-day work that IT people do isn’t directly related to delivering value. Serverless is a great mechanism for identifying waste.
+
+**Link of Interest:** [Contemporary Views on Serverless and Implications](https://m.subbu.org/contemporary-views-on-serverless-and-implications-1c5907c611d8)
+
+## Business Impacts
+
+If you’re non-technical then you are probably wondering why you should care about Serverless. After all, isn't it just another buzzword? Actually, Serverless is going to have more of an impact on how you work and operate your business than it will for the techies implementing it. Speed, visibility and agility is what it’s all about.
+
+Serverless is about speed. In my time at Mechanical Rock, I have personally received an end-to-end feature request (web, backing code and accompanying infrastructure) in the morning and delivered it to production fully tested by lunch time. A by-product of speed is the ability to experiment. Why try out just one idea, when it's possible to try out multiple ideas, gather the results and then implement the best bits?
```

Comment:
In all fairness, you could achieve that without serverless, so you perhaps need a comparison to other "cloud" methods around the economics.